### DISCUSSION TITLE
Microsoft Azure DevOps work items

### DISCUSSION THREAD

**Discussion by FloBec** [GitbookIO/community#29] [Feature Requests]
### What's your feature/idea?

Add a block element that resolves a Microsoft Azure DevOps work item id, showing id, type, name, status and as a bonus, an expandable button to view the description.

![image](https://user-images.githubusercontent.com/47239215/214058493-6fad6923-2bb8-47f4-b1f1-09d939572aee.png)

### What's your desired outcome?

We design/develop in Microsoft Azure DevOps and want to link work items and our documentation.

### What's the impact of this for you?

- [X] Critical
- [ ] High
- [ ] Medium
- [ ] Low

**Comment by rodrcastro** [comment]
Hey Florian! Thank you for your feedback :) Just to be clear, is your use case more focused on an internal knowledge base for your team, or are you sharing a page and listing out the work items there?

One thing you could try right now is grabbing a share link provided by the tool you are using and pasting it inside GitBook and hitting return. If we have support for embedding it already, it should do it right away. If not, let us know :)

**Comment by FloBec** [comment]
Hi Rodrigo,

we do ERP implementation projects with clients, but we also have our own industry solution to document. so its a bit of both: we document our industry solution and share it with clients to help them get to know the solution.

Microsoft Azure DevOps contains a Wiki, which allows embedded references to work items. We currently use that wiki, but would like to move on. Therefore my request aims toward feature parity with that Wiki.
